test/cassandra/container.spec.ts

```diff
@@ -0,0 +1,93 @@
+import "expect-puppeteer";
+import crypto from 'crypto'
```

Comment:
nit: `"`. We seem to use `"` to enclose string in Data Explorer. I wonder if our lint and format build targets can apply to these files as well.

Reply:
Yeah, these tests fail lint and are formatted but it's never changed the quotes or semicolons. Somewhat frustrating, maybe I can fix the lint rules